consensus/core/src/hashing/sighash.rs

```diff
@@ -64,6 +64,10 @@ impl<T: SigHashReusedValues> SigHashReusedValues for Arc<T> {
     fn outputs_hash(&self, set: impl Fn() -> Hash) -> Hash {
         self.as_ref().outputs_hash(set)
     }
+
+    fn payload_hash(&self, set: impl Fn() -> Hash) -> Hash {
+        self.as_ref().outputs_hash(set)
```

Comment:
oops this looks like a glitch `outputs_hash` -> `payload_hash`

Reply:
I see this `impl for Arc` is never used anyway, so perhaps we should just delete it